main.py

```diff
@@ -0,0 +1,136 @@
+from os import sched_get_priority_min
+from app.Tasklist import TaskList
```

Comment:
Python naming conventions say that your files/modules should be `snake_case`:

```suggestion
from app.task_list import TaskList
```